apps/console/src/features/sms-providers/pages/sms-providers.tsx

```diff
@@ -121,6 +122,20 @@ const SMSProviders: FunctionComponent<SMSProviderPageInterface> = (
     } = useSMSProviders();
     const [ isLoading, setIsLoading ] = useState(true);
 
+    const [ existingSMSProviders, setExistingSMSProviders ] = useState<string[]>([]);
+
+    useEffect(() => {
+        if(!isSMSProviderConfigFetchRequestLoading && originalSMSProviderConfig?.length > 0) {
+            const existingSMSProviderNames: string[] = [];
+
+            originalSMSProviderConfig?.map((smsProvider: SMSProviderAPIResponseInterface) => {
+                existingSMSProviderNames.push(smsProvider.provider + "SMSProvider");
+            });
```

Comment:
```suggestion
            const existingSMSProviderNames: string[] = originalSMSProviderConfig?.map((smsProvider: SMSProviderAPIResponseInterface) => {
                return existingSMSProviderNames.push(smsProvider.provider + "SMSProvider");
            });
```
Since `map()` function is used, better to update as above. Otherwise use `forEach()` function.

Reply:
Fixed with d735d9e